serverlessworkflow/modules/ROOT/pages/persistence/persistence-with-postgresql.adoc

```diff
@@ -1 +1,166 @@
-//= Persistence with PostgresSQL databases
\ No newline at end of file
+Running Serverless Workflow service using PostgresSQL
+=====================================================
+:compat-mode!:
+// Metadata:
+:description: Running Serverless Workflow service using PostgresSQL
+:keywords: kogito, workflow, quarkus, serverless, quarkus-cli, persistence, postgresql
+// Referenced documentation pages.
+:getting_started_create_first_workflow_guide: xref:getting-started/create-your-first-workflow-service.adoc
+:persistence_with_postgresql_guide: xref:persistence/persistence-with-postgresql.adoc
+// External pages
+:quarkus_datasource_guide: https://quarkus.io/guides/datasource
+:quarkus_datasource_devservices_guide: https://quarkus.io/guides/databases-dev-services
+:kogito_sw_callback_example: {kogito_sw_examples_url}/serverless-workflow-callback-quarkus
+:postgresql_url: https://www.postgresql.org/
+:postgresql_doc_url: https://www.postgresql.org/docs/current/
+
+This document describes how to run your Kogito Serverless Workflow application using PostgreSQL persistence.
+
+.Prerequisites
+* A Kogito Serverless Workflow service. More information available in {getting_started_create_first_workflow_guide}[Creating your first Serverless Workflow service] guide.
```

Comment:
```suggestion
include::../../pages/_common-content/getting-started-requirement.adoc[]
```

Reply:
Should we mention Java and Maven there in the common one, or create a new one?

Reply:
We already have Java and Maven mentioned in the getting started guide.